src/App.js

```diff
@@ -1,5 +1,14 @@
-function App() {
-	return <div className="App">Hello Time Divider</div>
+import GlobalStyle from './styles/GlobalStyle'
+import AppRouter from './routes/router'
+import { RecoilRoot } from 'recoil'
+
+const App = () => {
+	return (
+		<RecoilRoot>
+			<GlobalStyle />
+			<AppRouter />
+		</RecoilRoot>
+	)
 }
```

Comment:
P5;

소소한 JS팁 (중요하진 않음)

JS 함수에서 별도의 로직 없이 바로 값을 return 할 때에는 아래와 같이 생략이 가능해요!

```suggestion
const App = () => (
	<RecoilRoot>
		<GlobalStyle />
		<AppRouter />
	</RecoilRoot>
);
```

Reply:
`return`을 제거하니 코드가 훨씬 깔끔해서 commit suggestion을 누르고 싶어지네요 🙇‍♂️ 👍  
작업물에 반영해두도록 하겠습니다 ㅎㅎㅎ :)